happyrabbit/core/accounts/admin.py

```diff
@@ -0,0 +1,31 @@
+import nested_admin.nested
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import User
```

Comment:
Yernar, could you please get rid of `core` module and move everything under `user` or `hr_user` module